test/project_types/extension/commands/create_test.rb

```diff
@@ -5,18 +5,19 @@
 module Extension
   module Commands
     class CreateTest < MiniTest::Test
-      include TestHelpers::FakeUI
```

Comment:
Have you run the tests locally? I'm assuming that taking this out will cause unwanted output on the CLI during test runs. That's at least what happened with the push test. If there is no unwanted output, you're good to remove `FakeUI`.

Reply:
There was no unwanted output. I didn't see a difference in the output for when it removed or added.